_source/_includes/tracing-shipping/otel-troubleshooting.md

```diff
@@ -0,0 +1,171 @@
+This section contains some guidelines for handling errors that you may encounter when trying to collect traces with OpenTelemetry.
+
+## Problem: No traces are sent
+
+The code has been instrumented, but the traces are not being sent.
+
+### Possible cause - Collector not installed
+
+The OpenTelemetry collector may not be installed on your system.
+
+#### Suggested remedy
+
+Check if you have an OpenTelemetry collector installed and configured to receive traces from your hosts.
+
+
+### Possible cause - Collector path not configured
+
+If the collector is installed, it may not have the correct endpoint configured for the receiver.
+
+#### Suggested remedy
+
+1. Check that the configuration file of the collector lists the following endpoints:
+
+   ```yaml
+   receivers:
+     jaeger:
+       protocols:
+         thrift_compact:
+           endpoint: "0.0.0.0:6831"
+         thrift_binary:
+           endpoint: "0.0.0.0:6832"
+         grpc:
+           endpoint: "0.0.0.0:14250"
+         thrift_http:
+           endpoint: "0.0.0.0:14268"
+     opencensus:
+       endpoint: "0.0.0.0:55678"
+     otlp:
+       protocols:
+         grpc:
+           endpoint: "0.0.0.0:4317"
+         http:
+           endpoint: "0.0.0.0:4318"
+     zipkin:
+       endpoint: "0.0.0.0:9411"
+   ```
+
+2. In the instrumentation code, make sure that the endpoint is specified correctly. Refer to our [tracing documentation](https://app.logz.io/#/dashboard/send-your-data/collection?tag=all&collection=tracing-sources) for more on this.
+
+
+### Possible cause - Traces not genereated
+
+If the collector is installed and the endpoints are properly configured, the instrumentation code may be incorrect.
+
+
+#### Suggested remedy
+
+
+1. Check if the instrumentation can output traces to a console exporter.
+2. Use a web-hook to check if the traces are going to the output.
+3. Use the metrics endpoint of the collector (http://localhost:8888/metrics) to see the number of spans received per receiver and the number of spans sent to the Logz.io exporter.
+
+If the above steps do not work, refer to our [tracing documentation](https://app.logz.io/#/dashboard/send-your-data/collection?tag=all&collection=tracing-sources) and re-instrument the application.
+
+
+### Possible cause - Wrong exporter/protocol/endpoint
+
+If traces are generated but not send, the collector may be using incorrect exporter, protocol and/or endpoint.
```

Comment:
## List of endpoints to send data from your instrumented application
#### jaeger-protocols:
  - thrift_compact: "collector-url:6831"
  - thrift_binary: "collector-url:6832"
  - grpc: "collector-url:14250"
  - thrift_http: "collector-url:14268/api/traces"
#### opencensus: 
- "collector-url:55678"
#### otlp-protocols:
  - grpc: "collector-url:4317"
  - http: "collector-url:4318/v1/traces"
#### zipkin: 
- "collector-url:9411/api/v2/spans"